src/EnergyPlus/HVACManager.cc

```diff
@@ -2429,6 +2430,7 @@ void ReportAirHeatBalance(EnergyPlusData &state)
         state.dataHVACMgr->ReportAirHeatBalanceFirstTimeFlag = false;
     }
 
+    ReportInfiltrations(state);
```

Comment:
Update object-level reporting variables right before zone level reporting, to ensure consistency